src/contract.rs

```diff
@@ -65,18 +65,29 @@ pub fn query<S: Storage, A: Api, Q: Querier>(
         QueryMsg::Holder { address } => to_binary(&query_holder(&deps, address)?),
         QueryMsg::Holders { start_after, limit } => {
             to_binary(&query_holders(&deps, start_after, limit)?)
-        }
+        },
+        QueryMsg::TransferFrom { .. } => to_binary(&query_transfer_from(deps)?),
     }
 }
 
+fn query_transfer_from<S: Storage, A: Api, Q: Querier>(
+    _deps: &Extern<S, A, Q>,
+) -> StdResult<StdError> {
+    Err(StdError::Unauthorized { backtrace: None })
+}
+fn query_transfer<S: Storage, A: Api, Q: Querier>(_deps: &Extern<S, A, Q>) -> StdResult<StdError> {
+    Err(StdError::Unauthorized { backtrace: None })
+}
+
 fn query_config<S: Storage, A: Api, Q: Querier>(
     deps: &Extern<S, A, Q>,
 ) -> StdResult<ConfigResponse> {
     let config: Config = read_config(&deps.storage)?;
     Ok(ConfigResponse {
-        lottery_contract: deps.api.human_address(&config.lottery_contract)?,
-        cw20_token_contract: deps.api.human_address(&config.cw20_token_contract)?,
+        admin: deps.api.human_address(&config.admin)?,
```

Comment:
Why removed external lottery contract address?

Reply:
There is no need on the staking contract of the lottery address